components/assetstore-controller-manager/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Build the manager binary
+FROM golang:1.10.3 as builder
+
+# Copy in the go src
+WORKDIR /go/src/github.com/kyma-project/kyma/components/bucket-controller
+COPY pkg/    pkg/
+COPY cmd/    cmd/
+COPY vendor/ vendor/
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager github.com/kyma-project/kyma/components/bucket-controller/cmd/manager
+
+# Copy the controller-manager into a thin image
+FROM ubuntu:latest
```

Comment:
Please consider using a different base image (like alpine or scratch) and definitely do not use the latest tag.